### ISSUE TITLE
Implement the search for international results

### ISSUE THREAD

**Issue by LeDrieHda** [hochschule-darmstadt/einander-helfen#247]
**Reason (Why?)**
The usefulness of einander-helfen.org depends on the versatility of the results. We want to be able to offer international, as well as national, results.

**Solution (What?)**
The user should be able to switch between national and international search. They should be able to search for countries and/or continents, as well as do a full-text search of the international posts, either within a selected country/continent or without selecting one to show all international results relevant to their search term.

**Out of scope**
The search for the national results stays as is. The international search is merely an extension to that.
Layout/UI will be done in another (frontend) issue.

**Relation to other Issues**
#224 
#244 has to be done before implementing this issue.

**Acceptance criteria**
When international search ("Weltweit" in the dropdown next to the location search bar) is selected and the user starts typing in the location search bar, autocomplete should offer countries or continents instead of postal code and place. Any given search results in "international mode" should be international posts only, located anywhere but in Germany. 
Searching for a country or continent returns posts located in that country/continent. Full-text search without selecting a country/continent returns all fitting international results. Full-text search with a country/continent selected returns all fitting results in the respective country/continent.

**Comment by DianaPankratz** [comment]
* Correct select size in tablet view
* Edit smartphone header
* Don't use radius in interntaional search
